src/execution_plan/ops/op_traverse.c

```diff
@@ -76,7 +76,7 @@ OpResult TraverseConsume(OpBase *opBase, Record *r) {
     Node *srcNode = Graph_GetNode(op->graph, src_id);
     Node *destNode = Graph_GetNode(op->graph, dest_id);
     Record_AddEntry(r, op->algebraic_results->src_node->alias, SI_PtrVal(srcNode));
-    Record_AddEntry(r, op->algebraic_results->dest_node->alias, SI_PtrVal(srcNode));
+    Record_AddEntry(r, op->algebraic_results->dest_node->alias, SI_PtrVal(destNode));
```

Comment:
How didn't we noticed that before, I guess op_traverse isn't that popular

Reply:
@swilly22 Huh, I just looked into this and at the moment, op_traverse isn't even possible. They can only be introduced by the `reduceScans` execution plan optimization, but the call to that function is commented out (`optimer.c:20`).

I'm not sure what's going on there, do you remember?